docs/ops/logical/LogicalAnd_1.md

```diff
@@ -6,39 +6,40 @@
 
 **Short description**: *LogicalAnd* performs element-wise logical AND operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**: Before performing logical operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *LogicalAnd* does the following with the input tensors *a* and *b*:
+
+\f[
+o_{i} = a_{i} and b_{i}
+\f]
```

Comment:
The current math formula is rendered like:
![image](https://user-images.githubusercontent.com/20084299/127467471-133b1e01-b70b-450f-b2de-82eef8e0ee8d.png)
I believe we can make it more readable.

Reply:
```suggestion
\f[
o_{i} = a_{i} \wedge b_{i}
\f]
```
I think this might look a bit better

Reply:
Updated the math formula